src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/thermoPoroElasticWellbore/Example.rst

```diff
@@ -0,0 +1,252 @@
+
+and
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/THM_wellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_HeatCapacityProperties -->
+  :end-before: <!-- SPHINX_HeatCapacityPropertiesEnd -->
+
+The permeability of rock is defined by:
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/THM_wellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_PermeabilityProperties -->
+  :end-before: <!-- SPHINX_PermeabilityPropertiesEnd -->
+
+Fluid properties such as viscosity, thermal expansion coefficient, etc. are defined by the XML block below. A negligible volumetric heat capacity is defined for fluid to ignore the thermal convection effect such that only thermal transfer via the diffusion phenomenon is considered.
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/THM_wellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_FluidProperties -->
+  :end-before: <!-- SPHINX_FluidPropertiesEnd -->
+
+-----------------------------------------------------------
+Boundary conditions
+-----------------------------------------------------------
+
+The mechanical boundary conditions are applied to ensure the axisymmetric plane strain conditions such as:
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/THM_wellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_PlaneStrainAxisymmetryBC -->
+  :end-before: <!-- SPHINX_PlaneStrainAxisymmetryBCEnd -->
+
+Besides, the far-field boundary is assumed to be fixed because the local changes on the wellbore must have negligible effect on the far-field boundary. 
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/THM_wellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_FixedFarFieldBC -->
+  :end-before: <!-- SPHINX_FixedFarFieldBCEnd -->
+
+The traction-free condition on the inner surface of the wellbore is defined by:
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/THM_wellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_ZeroInnerTractionBC -->
+  :end-before: <!-- SPHINX_ZeroInnerTractionBCEnd -->
+
+The initial temperature (that is also the far-field boundary temperature) and the temperature applied on the inner surface of the wellbore are defined as
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/THM_wellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_TemperatureBC -->
+  :end-before: <!-- SPHINX_TemperatureBCEnd -->
+
+It is important to remark that the initial stress of each layers must be set with accordance to the initial temperature: :math:`\sigma_{0} = 3K\alpha T_{0}` where :math:`\sigma_{0}` is the initial principal stress, :math:`T_{0}` is the initial temperature, :math:`K` is the drained bulk modulus and :math:`\alpha` is the drained linear thermal expansion coefficient of the materials. In this example, the initial stresses are set to zero because the initial temperature is set to zero (°C).
```

Comment:
```suggestion
It is important to remark that the initial effective stress of each layers must be set with accordance to the initial temperature: :math:`\sigma_{0} = 3K\alpha T_{0}` where :math:`\sigma_{0}` is the initial principal effective stress, :math:`T_{0}` is the initial temperature, :math:`K` is the drained bulk modulus and :math:`\alpha` is the drained linear thermal expansion coefficient of the materials. In this example, the initial effective stresses are set to zero because the initial temperature is set to zero degrees Kelvin.
```
Maybe just say that yes 0 K may seem to be a cold temperature, but `T` (and `p`) should be interpreted here as delta T and delta P with respect to an initial equilibrium.